src/app/services/users.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { environment } from '../../environments/environment';
+import { User } from '../models/User';
+
+const httpOptions = {
+  headers: new HttpHeaders({ 'Content-Type': 'application/json' }),
+};
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UsersService {
+  constructor(private http: HttpClient) { }
+  tagId:number;
+
+  getExpertTags(): Observable<any[]>{
+    return this.http.get<any[]>(`${environment.userUri}`);
+  }
```

Comment:
remove state, and why is this in a UsersService?

Reply:
changed to getExperts